docs/btnews/0801_0900/btnews_0882.md

```diff
@@ -0,0 +1,321 @@
+And also I work as consultant of provisional provincial and cities government in China.
+This is at a professional level.
+I I feel very very close to to China.
+In 2010, I live in Shanghai, the whole year.
+Because this organization that I told you before, Devnet, has its own pavilion in Shanghai World Expo in 2010.
+And this is when my book has been printed in China.
+
+我大多數觀眾的年齡在20歲到35歲之間，所以您第一次來中國的時候，他們還沒出生。
+您能不能從您的角度介紹一下您觀察中國這些年有什麼變化？
+
+My first time, as I told you, in China has been in 1989.
+Then I had the chance and the honor to meet Mr. Deng Xiaoping.
+And also we had been invited to Shanghai.
+And the then mayor of Shanghai, that then became Mr. Zhu, that then he became first Minister of China, invite us to visit Shanghai.
+And I remember that he describe or told us about which were the plans to develop Shanghai.
+And he show us which were the plans to build, for example, now is Pudong.
+And believe me, that in that time, what your audience knows as Pudong were rice plants.
+Very hard to to achieve this in 10 years.
+My second time was in 2000, 11 years later.
+And when I took a flight from Rome to Shanghai, and when I land in Shanghai, and I saw what was Pudong, I think I took the wrong flight, because I can't believe that in 11 years, all that the mayor say that he will develop in Shanghai, had been reached, has been done.
+Your audience, the young Chinese people, should know what is called the Chinese miracle.
+Because, and this is not only my opinion, what China has done in 40 years, no country in the history, in the human history has done.
+
+您剛才提到您第一次來中國就見到了鄧小平。
+您對他個人有什麼記憶？
+
+He received us, he host us in the People, in Tiananmen Square, People Palace.
+And he smoked a lot.
+And he said that I have a nightmare.
+How how to give to all the Chinese people the possibility to have two meals a day, a day.
+And he say, China will never, never take the same road of reform as the the Soviet Union has walked by that road.
+And I feel that he has a very strategic vision about how to solve the problem of China.
+About how to give all the people the chance to eat two times a day as the minimum thing that you have to give to those people.
+And as I told before, the results, and this is why I think that Mr. Deng Xiaoping is the in the last century, the most relevant statesman.
+
+### 嘉賓關於中國的著作
+
+聽說您已經寫了一本關於中國的書，而且還在寫第二本。
+我能夠請您介紹一下您下一本書想表達的內容嗎？
+因為我們畢竟絕大多數中國人都沒有體驗過從拉丁美洲或者從南美洲來看中國的感受，我猜有很多觀眾想讀這本書。
+
+Yes, as you say, in 2010, it's been 15 years ago, I wrote my first book about China.
+And I call this book China breathes a new era.
+And this book has been printed here in China.
+Why I title my book China breathes a new era?
+Because China has is changing but has already changed the world.
+And in my and China breathes a new era for the not only for China, but for the world.
+Now, this book that I am already finished to write it, is called, the title is Discovering China.
```

Comment:
```suggestion
Why I title my book China breathes a new era?
Because China has is changing but has already changed the world.
And in my and China breathes a new era for the not only for China, but for the world.
Now, this book that I am already finished to write it, is called, the title is Discovering China.


**为什么我将这本书命名为《中国孕育新时代》？**

因为中国正在改变，而且已经改变了世界。而且在我看来，中国正在为不仅是中国，也为世界孕育新时代。现在，我已经写完的这本书，书名叫《发现中国》。

```